src/search_results.css

```diff
@@ -1,10 +1,15 @@
 .highlighted-search-result {
-  /* background-color: rgba(255, 49, 49, 0.3); */
+	/* nothing */
 }
 
-.highlighted-search-result::before {
+.unhighlighted-search-result, .unhighlighted-search-result:focus  {
+  outline: none;
+}
+
+.highlighted-search-result::before,
+.unhighlighted-search-result::before {
   content: "\25BA";
   margin-right: 25px;
   left: -25px;
   position: absolute;
-}
+}
```

Comment:
I think `highlighted-search-result` and `unhighlighted-search-result` is a bit confusing. Perhaps rename the former to `focused-search-result` and the latter to `no-outline`?
Then always add the `focused-search-result` class to the focused element, and conditionally add the `no-outline` class if the option is set.